DOCS/man/options.rst

```diff
@@ -5557,13 +5557,12 @@ them.
         Automatic selection.
         On ``vo=gpu``: if output bit depth cannot be detected, 8 bpc is assumed.
         On ``vo=gpu-next``: with ``gpu-api=d3d11``, real on-the-wire bpc is used.
-        For other ``gpu-api``, 8 bpc is used for SDR content.
+        For ``gpu-api=vulkan``, 8 bpc is assumed for SDR content.
```

Comment:
I think we should indeed simplify it. Make it more vague and say that it is best effort to provide best depth depending on vo and gpu-api. Exactly wording is up to you, but describing full logic here make it hard to parse in docs imho.